packages/sourcecred/package.json

```diff
@@ -130,8 +130,7 @@
     "flow": "flow",
     "lint": "eslint src config --max-warnings 0",
     "prepublishOnly": "yarn build",
-    "docs": "node ./scripts/build-docs.js",
-    "postinstall": "../../scripts/flow-mono.sh"
```

Comment:
oof my bad. this shouldn't have gotten merged

Reply:
What's the game plan here